### DISCUSSION TITLE
how to click on icon

### DISCUSSION THREAD

**Discussion by anaschaudhary33** [getgauge/taiko#2655] [General]
how can I click on this (+) icon below operation-
[](url)
![Screenshot from 2022-10-31 15-51-16](https://user-images.githubusercontent.com/70742968/198987298-75d10e20-f341-4021-b579-9c69429c5c30.png)

its CSS selector:
```
td.align-center > span:nth-child(1)
```
Xpath
```
 /html/body/div/section/div/article/div/section/form[1]/fieldset/div[2]/table/thead/tr[2]/td[3]/span
```

I can click on this icon using puppeteeer
```
await page.click('.table > #macFilterHead > tr > .align-center > .ico')
```

**Comment by jojurko** [comment]
await click($("td.align-center > span:nth-child(1)"))
